runtime/error_test.go

```diff
@@ -178,6 +178,66 @@ func TestRuntimeError(t *testing.T) {
 		)
 	})
 
+	t.Run("execution multiline nested error", func(t *testing.T) {
+
+		t.Parallel()
+
+		runtime := newTestInterpreterRuntime()
+
+		script := []byte(`
+			pub resource R {
+				init(s:String){
+					panic("42")
+				}
+			}
+		
+			pub fun createR(): @R{
+				return <- create R(
+					s: "argument"
+				)
+			}
+			
+			pub fun main() {
+				destroy createR()
+			}
+        `)
+
+		runtimeInterface := &testRuntimeInterface{}
+
+		location := common.ScriptLocation{0x1}
+
+		_, err := runtime.ExecuteScript(
+			Script{
+				Source: script,
+			},
+			Context{
+				Interface: runtimeInterface,
+				Location:  location,
+			},
+		)
+
+		require.EqualError(t, err,
+			"Execution failed:\n"+
+				"  --> 0100000000000000000000000000000000000000000000000000000000000000:15:12\n"+
+				"   |\n"+
+				"15 | 				destroy createR()\n"+
+				"   | 				        ^^^^^^^^^\n"+
+				"\n"+
+				"  --> 0100000000000000000000000000000000000000000000000000000000000000:9:21\n"+
+				"   |\n"+
+				" 9 | 				return <- create R(\n"+
+				"10 | 					s: \"argument\"\n"+
+				"11 | 				)\n"+
+				"   | 				^^^^^^^^^^^^^^^^^^\n"+
```

Comment:
It looks like the indicator is short of once character. Not quite sure, but is there maybe an "off-by-one" bug here?

Reply:
This isn't actually an off-by-one, it's an off by `len(name of resource)` error here that just looks like an off-by-one because the name here is `R`! Nice catch. Should be fixed now.